### ISSUE TITLE
Hi , can I train this model on my own dataset ?

### ISSUE THREAD

**Issue by jediofgever** [WangYueFt/dgcnn#41]

**Comment by WangYueFt** [comment]
You probably need to do two things:

1. implement a new dataloader or modify the existing one.
2. change the input placeholder to be compatible with your data.

**Comment by jediofgever** [comment]
@WangYueFt  , Thank you for your reply,  Also , if my point cloud contains more than say 2048 points, what will be expected behaviour of network to it , do I need to have strictly 2048 points  ?

**Comment by WangYueFt** [comment]
> @WangYueFt , Thank you for your reply, Also , if my point cloud contains more than say 2048 points, what will be expected behaviour of network to it , do I need to have strictly 2048 points ?

You don't need to. But in each batch, you have to pad them into the same number of points.

**Comment by jediofgever** [comment]
@WangYueFt, I did create data files same as .h5 format and trained network but even in epoch 0 I got 0.99 train accuracy. 

`Train 0, loss: 1.376507, train acc: 0.993896, train avg acc: 0.993896`
`Test 0, loss: 1.233287, test acc: 1.000000, test avg acc: 1.000000`
`Train 1, loss: 1.233263, train acc: 1.000000, train avg acc: 1.000000`
`Test 1, loss: 1.233124, test acc: 1.000000, test avg acc: 1.000000`
`Train 2, loss: 1.233195, train acc: 1.000000, train avg acc: 1.000000`
`Test 2, loss: 1.233121, test acc: 1.000000, test avg acc: 1.000000`
`Train 3, loss: 1.233166, train acc: 1.000000, train avg acc: 1.000000`
`Test 3, loss: 1.233120, test acc: 1.000000, test avg acc: 1.000000`
`Train 4, loss: 1.233150, train acc: 1.000000, train avg acc: 1.000000`
`Test 4, loss: 1.233116, test acc: 1.000000, test avg acc: 1.000000`

Does this mean model if overfitting ?

**Comment by AdeV-Oly** [comment]
Hey there @jediofgever 

I'm attempting to train (the equivariant version (VNN) of) this model on my own dataset as well. I think I may be overlooking something, as I'm running into an error.

If you or @WangYueFt could take a look at it and give me some pointers, or if I need to add some more info, that'd be much appreciated.

Link to my issue on the VNN github: https://github.com/FlyingGiraffe/vnn/issues/13
